### ISSUE TITLE
use serial port to control a camera instead of usb (sony camera)

### ISSUE THREAD

**Issue by albuswu** [gphoto/gphoto2#263]
It turned out that after I connect my Sony A6000/A7II to the USB port (raspberry pi 3), the battery saving function will not work and the camera will be on forever until the battery gets drained. I am working on a time lapse project that will last a few weeks with 30mins interval. So the battery saving function is essential.

Is that possible to output serial commands through the USB-micro USB cable to trigger the camera? Just like what a normal camera remote do. This should solve the problem. I am looking into it right now but haven't found any solution yet.

**Comment by msmeissn** [comment]
I dont think this serial control is possible over a USB connection.

Is the Sony however not charging over the port?

**Comment by albuswu** [comment]
No. The older models don't have charging during operation... That's also the reason why I am trying gphoto2, but then it comes with this new problem. it's alright, maybe I can use GPIO for triggering, or get a new camera...
